notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -12041,6 +12041,51 @@ fear [There were pointed moment when Jillian was in terror of her husband, Spenc
+Peut-être
+
+:: Description
+Peut-être (Maybe; Perhaps) is a 1999 French science fiction comedy film. After
+having sex with his girlfriend Lucie in a bathroom, Arthur discovers that a
+ceiling panel is a time portal to the Paris in the future, although it appears
+more like a sun-baked desert city by that point. There he meets an old man
+named Ako who turns out to be Arthur's son. Ako attempts to persuade Arthur to
+impregnate Lucie so that he can exist in this future.
+
+:: Date
+1999-11-10
+
+:: References
+https://en.wikipedia.org/wiki/Peut-%C3%AAtre
+
+:: Ratings
+3 <paul>
+
+:: Choice Themes
+what if I met my distance descendants [The main novelty of the film concerns the young man Arthur meeting his future children and grandchildren after passing through a time portal.]
+father and son [The young man Arthur traveled into the future whereupon he met his future son, Ako, who was by then an old man. The story then follows Ako's attempts to persuade Arthur to impregnate his girlfriend so that Ako and his progeny can continue to exist in the future.]
+
+:: Major Themes
+what if I found myself in the future [Arthur passed through a time portal to find himself 60 years in the future. Not only did he find the Paris of the future to be a sun-baked desert, but he also met his future son and grandchildren.]
+personal practical dilemma [Should Arthur impregnate his girlfriend or not? If he did, then his future son Ako and Ako's descendants would go on living happily in the future; otherwise they would cease to exist.]
+facing death [Ako was faced with the prospect of vanishing into thin air should he fail in his attempts to convince his son, Arthur, to impregnate his mother, Lucie.]
+boyfriend and girlfriend [Arthur and his girlfriend Lucie found themselves in a most strange predicament while attending a wild, all-night millennium party.]
+
+:: Minor Themes
+time portal [Arthur discovered a bathroom ceiling panel was actually a time portal and subsequently passed through it to find himself about 60 years in the future.]
+husband and wife [Ako introduced Arthur to his wife Marie-Jeanne.]
+father and daughter [Ako introduced Arthur to Arthur's future daughter. Rosemonde tried to keep her crazy, gun-toting father under control.]
+brother and sister [Ako briefly quarreled with his sister Madeline.]
+illegal drug trade [The millennium party goes were shown smoking and snorting various street drugs.]
+grandfather and granddaughter [Ako conversed with his granddaughter while tied up in a chair.]
+what if I found myself in the past [Ako traveled back in time about 60 years and there met his mother, Lucie.]
+mother and son [Overcome with emotions, Ako wept as he danced with his mother, Lucie, at the millennium party.]
+gender bending [Kader was initially surprised when the girl he was hitting on at the millennium party revealed herself to be (in her words) "a boy", but this was not ultimately a problem for Kader, and they hooked up after the party.]
```

Comment:
A bit of male homosexuality featured too then?

Reply:
Did you see this comment?

Reply:
Ah, I missed this one. I'd think we'd do to coin a new theme, if we want to capture something about their interaction. For example, see

"Cisgender men attracted to transgender women primarily identify as heterosexual and sometimes as bisexual, but rarely as homosexual"

here <https://en.wikipedia.org/wiki/Attraction_to_transgender_people>.

Reply:
I see. Well I learned yesterday that although Hemingway was certainly not homosexual, he could be called https://en.m.wikipedia.org/wiki/Queer because of a reported penchant for gender role reversal in bed. "Queer" is an umbrella term for all non-cis sexualities I understand, so might make an excellent meta-theme for us.

Reply:
Ah yes, why that sounds very profitable, indeed. I'll leave it to you to define, if you wish, and I'll add it to this story with gusto thereafter.

Reply:
I review these from my phone so we'd better try and recall these excellent suggestions during the next meeting if we want them implemented!

Reply:
Pinned a message about it in the-themes Slack channel. Won't be losing sight of that anytime soon.